swf/settings.py

```diff
@@ -105,13 +105,16 @@ def from_env():
     :rtype: dict
 
     """
-    try:
-        return {'aws_access_key_id': os.environ['AWS_ACCESS_KEY_ID'],
-                'aws_secret_access_key': os.environ.get('AWS_SECRET_ACCESS_KEY'),
-                'region': os.environ.get('AWS_DEFAULT_REGION')
-        }
-    except KeyError:
-        return {}
+    hsh = {}
+
+    if "AWS_ACCESS_KEY_ID" in os.environ:
```

Comment:
Credentials can be fetched by boto, but not sure for the region, it has to be passed explicitly down the road, or "us-east-1" is used if I remember correctly.

Anyway yes: the idea here was to use "etcaetera", which provides flexible backends, overrides order and a clean fallback (we could try consul, then file, then env for instance). To my knowledge, we only use env variables, so it's useless. I'll try to remove it later if you agree.